hack/prune-libraries.sh

```diff
@@ -19,6 +19,10 @@
 
 # By default just checks for extra libraries, run with --fix to make changes.
 
+# TODO: rewrite this in Python, or fix this!
+# SC2086: Double quote to prevent globbing and word splitting.
```

Comment:
This is one of the more useful checks... we should fix the script, not do this

Reply:
There's really not much that can be done with this while maintaining OSX compatibility.

Reply:
Please explain?